hydrant/adapters/sites/skagit.py

```diff
@@ -54,3 +57,71 @@ def unique_key(self):
         comparison operators.
         """
         return json.dumps([self.name, self.birthDate])
+
+
+class SkagitServiceRequestAdapter(object):
+    """Specialized site adapter for skagit site service request exports"""
+    RESOURCE_CLASS = ServiceRequest
+    SITE_CODING_SYSTEM = "http://hl7.org/fhir/sid/icd-9"
+
+    @classmethod
+    def headers(cls):
+        """Return minimal expected header values - extras ignored"""
+        return [
+            'Test Code Ordered',
+            'Specimen Date',
+            'Pat Last Name',
+            'Pat First Name',
+            'Pat DOB',
```

Comment:
Should these be a class attribute or enum?

Reply:
Probably, someday.  It's still evolving, so far this simple pattern is working well.